src/commands/seek.ts

```diff
@@ -867,6 +867,117 @@ export async function leap(
   }
 }
 
+/**
+ * Goto word
```

Comment:
nit: I would call this "Leap to word" to match the "leap" naming. The description could mention that it achieves the same thing as the Helix `g w` command though.